lib/run-status.js

```diff
@@ -134,9 +135,10 @@ RunStatus.prototype.prefixTitle = function (file) {
 	var separator = ' ' + chalk.gray.dim(figures.pointerSmall) + ' ';
 
 	var prefix = path.relative('.', file)
-		.replace(this.base, '')
+		.replace(new RegExp('^' + escapeStringRegexp(this.base)), '')
```

Comment:
@novemberborn to avoid the regex, how about this?
```js
.replace(this.base, function (match, offset, string) {
	if (offset === 0) {
		return ''
	} else {
		return match
	}
})
```